pkg/registry/fetch.go

```diff
@@ -59,10 +65,31 @@ func (f *Fetcher) Fetch(dir, registryImage string) error {
 		handler = handleSource
 	}
 
-	if err := handler(img, dir); err != nil {
+	if err = handler(img, dir); err != nil {
 		return err
 	}
 
+	projectMetadataFile, err := os.Create(path.Join(metadataDir, "project-metadata.toml"))
+	if err != nil {
+		return errors.Wrapf(err, "invalid metadata destination '%s/project-metadata.toml' for image: %s", metadataDir, registryImage)
```

Comment:
We're using `errors.Wrap` where and not `fmt.Errorf`.  When do we wrap and when we `Errorf`?